charts/k8ssandra/templates/grafana/cassandra-condensed.configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.monitoring.grafana.provision_dashboards }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-k8ssandra-grafana-cassandra-condensed-dashboard
```

Comment:
What do you think about shortening the name to `{{ .Release.Name }}-cassandra-condensed`?

Reply:
I'd want to keep some sort of indicator that it's for grafana and / or a dashboard.

WDYT?

1. `{{ .Release.Name }}-k8ssandra-grafana-cassandra-condensed-dashboard`
2. `{{ .Release.Name }}-cassandra-condensed`
3. `{{ .Release.Name }}-grafana-cassandra-condensed`
4. `{{ .Release.Name }}-dashboard-cassandra-condensed`
5. `{{ .Release.Name }}-grafana-dashboard-cassandra-condensed`

Reply:
Note I tried to go from least to most specific, that may be foolish, but it's a convention :shrug:

Reply:
Thanks for laying out the options. I guess I'd go with 5.